.travis.yml

```diff
@@ -1,25 +1,28 @@
 language: node_js
 matrix:
   include:
-    - node_js: "6"
-    - node_js: "7"
-      env: RUN_RECIPES=1
-  allow_failures:
-    - node_js: "7"
+    - node_js: "8"
+    - node_js: "9"
       env: RUN_RECIPES=1
 dist: trusty
 cache:
   yarn: true
   directories:
     - node_modules
-before_install:
-  - npm install -g --no-progress yarn@v0.22.0
+    - lighthouse-extension/node_modules
+    - lighthouse-viewer/node_modules
+    - /home/travis/.rvm/gems/
```

Comment:
the two lighthouse-* folders dont make sense to me here.
and i dont think you're running ruby, so you can nuke this one too.